### PR TITLE
ADDED SAMPLE APPLICATION

### LINKED ISSUE #1490: [Docs]  List supported operations for the Meshery adapter for Octarine
**Current State:**

The operations supported by the adapter for Linkerd are not listed in its [documentation](https://meshery.layer5.io/docs/service-meshes/adapters/octarine/octarine).

**Desired State:**

Under the heading `Sample Applications` and include a list of the supported operations, with a small explanation for each. The operations can be found listed [here](https://github.com/layer5io/meshery-octarine/blob/master/octarine/supported_ops.go); (lines 34-54). Look out for the following line: `opType: meshes.OpCategory_SAMPLE_APPLICATION`. This means that the operation listed is a sample application.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by was-coder**
I added the book info sample application to the octarine documentation

Signed-off-by: was-coder <wasleadwears4u2c@gmail.com>

**Description**

This PR fixes #1490 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by KarenEfereyan** [comment]
Nice work

**Comment by was-coder** [comment]
Thanks Karen 😊

File: docs/_adapters/octarine/octarine.md [hunk 1]
```diff
@@ -60,3 +60,10 @@ Once the Octarine's data plane services are deployed, the adapter can be used to
 
 - Examine [Meshery's architecture]({{ site.baseurl }}/architecture) and how adapters fit in as a component.
 - Learn more about [Meshery Adapters]({{ site.baseurl }}/architecture/adapters).
+
+### Sample Applications 
+
+The Meshery adapter for {{ page.name }} includes the below sample application operation. Meshery can be use to deploy this sample application.
+
+- [Istio BookInfo](https://github.com/layer5io/istio-service-mesh-workshop/blob/master/lab-2/README.md#what-is-the-bookinfo-application)
```
Comment (leecalcote) [inline docs/_adapters/octarine/octarine.md:68]: Whoa! Nice, @was-coder 👍

**Comment by was-coder** [comment]
Thanks @leecalcote

**Pull request merged**